.gitignore

```diff
@@ -11,3 +11,4 @@ temp
 *.tfstate
 *.tfstate.backup
 /ssh-bastion.conf
+**/*.sw[pon]
```

Comment:
``**/`` will cover all directory below as well, it's recursive. https://git-scm.com/docs/gitignore